.github/workflows/ci.yml

```diff
@@ -12,36 +12,30 @@ on:
 jobs:
   test:
     name: "PHPUnit: MW ${{ matrix.mw }}, PHP ${{ matrix.php }}"
+    continue-on-error: ${{ matrix.experimental }}
 
     strategy:
+      fail-fast: false
```

Comment:
Why set this to false? (In other PR's too.) This basically overrides the purpose of `continue-on-error`. If we're not going to stop CI when a non-experimental job fails, then there's no point in setting the experimental flag.